apps/src/blockly/addons/cdoFieldVariable.js

```diff
@@ -51,11 +51,12 @@ export default class FieldVariable extends GoogleBlockly.FieldVariable {
       }
     }
   }
+  menuGenerator_ = newDropdownCreate;
 }
 
-FieldVariable.originalDropdownCreate = FieldVariable.dropdownCreate;
-FieldVariable.dropdownCreate = function() {
-  const options = FieldVariable.originalDropdownCreate.call(this);
+const originalDropdownCreate = FieldVariable.dropdownCreate;
+const newDropdownCreate = function() {
+  const options = originalDropdownCreate.call(this);
```

Comment:
now that we aren't replacing fields on the `FieldVariable` class, i feel like we lose some of the intent with this approach (setting some `const`s here and setting `menuGenerator_` elsewhere). would it work to do something similar, but with a named function (prototype of what i mean below)?

```javascript
class FieldVariable extends GoogleBlockly.FieldVariable {
  onItemSelected_(menu, menuItem) {
    menuGenerator_ = this.newDropdownCreate;
  }

  newDropdownCreate() {
    const options = FieldVariable.dropdownCreate.call(this);
    // ... rest of function
  }
}
```

caveat: i might've gotten something wrong with scoping and what `this` is in my prototype function, but what do you think of that?